draft-irtf-panrg-path-properties.md

```diff
@@ -124,6 +124,13 @@ Assessed property:
 : An approximate calculation or assessment of the value of a property. An assessed property includes the reliability of the calculation or assessment. The notion of reliability depends on the property.
 For example, a path property based on an approximate calculation may describe the expected median one-way latency of packets sent on a path within the next second, including the confidence level and interval. A non-numerical assessment may instead include the likelihood that the property holds.
 
+## Terminology usage for specific technologies
+
+The terminology described in this document is intended to be general and applicable to existing and future path-aware technologies.
```

Comment:
```suggestion
The terminology defined in this document is intended to be general and applicable to existing and future path-aware technologies.
```
I think "define terminology" is more fitting than "describe terminology" in this context. (Also, I'm getting 42.5k Google hits for "define terminology" and just 5.5k for "describe terminology" :smile:)

Reply:
Agreed ;)